### ISSUE TITLE
Php tests failing in `phpunit-php7.4-sqlite-owncloud` pipelines

### ISSUE THREAD

**Issue by SwikritiT** [owncloud/core#40436]
php tests are failing in `phpunit-php7.4-sqlite-owncloud` pipeline for all the builds recently 

All the tests are failing with `OCP\Files\StorageNotAvailableException: Sabre\HTTP\ClientHttpException: Bad Request`
Build link : https://drone.owncloud.com/owncloud/core/37065/27/9

```
There were 85 errors:

1) OCA\Files_External\Tests\Storage\OwncloudTest::testRoot
OCP\Files\StorageNotAvailableException: Sabre\HTTP\ClientHttpException: Bad Request

/drone/src/lib/private/Files/Storage/DAV.php:902
/drone/src/lib/private/Files/Storage/DAV.php:860
/drone/src/lib/private/Files/Storage/DAV.php:695
/drone/src/lib/private/Files/Storage/DAV.php:175
/drone/src/apps/files_external/tests/Storage/OwncloudTest.php:53
phpvfscomposer:///drone/src/lib/composer/phpunit/phpunit/phpunit:97
```

**Comment by SwikritiT** [comment]
@phil-davis do you have any idea why this might be happening?

**Comment by phil-davis** [comment]
`owncloudci/php` now has a daily cron job that rebuilds it, so that any upstream changes to base Ubuntu and ownCloud docker images are picked up quickly (and any new patch releases of PHP etc)

https://hub.docker.com/layers/owncloudci/php/7.4/images/sha256-f847c657ab825e19915ab84b9295f1537d7a45831d13d9d184a80f1f7f6f399f?context=explore

I guess "something happened" (tm) - now I will think about what might have changed that would break some external storage tests.

**Comment by phil-davis** [comment]
Note: failed in last night's Transifex merge CI https://drone.owncloud.com/owncloud/core/37058/23/9

Passed in a full-ci PR drone run yesterday: https://drone.owncloud.com/owncloud/core/37051/27/9

So "something broke" late yesterday.

**Comment by phil-davis** [comment]
> So "something broke" late yesterday.

The change yesterday was that the owncloud server docker image now requires that OWNCLOUD_TRUSTED_DOMAINS be set. See https://central.owncloud.org/t/important-changes-to-owncloud-server-container-deployments/39642

Fixed by #40438
